bamMergePreprocessing.wdl

```diff
@@ -9,32 +14,30 @@ struct GenomeResources {
 workflow bamMergePreprocessing {
 
   input {
-    Array[InputGroup] inputGroups
+    Array[bamFiles] inputBamFiles
+    String outputFileNamePrefix
     String intervalsToParallelizeByString
     Boolean doFilter = true
     Boolean doMarkDuplicates = true
-    Boolean doSplitNCigarReads = false
-    Boolean doIndelRealignment = true
-    Boolean doBqsr = true
+    Boolean doBqsr = false
     String reference
     String reference_genome
   }
 
   parameter_meta {
-    inputGroups: "Array of objects describing sets of bams to merge together and the merged file name. These merged bams will be cocleaned together and output separately (by merged name)."
+    inputBamFiles: "Array of objects describing sets of bams to merge together and the merged file name. These merged bams will be cocleaned together and output separately (by merged name)."
     intervalsToParallelizeByString: "Comma separated list of intervals to split by (e.g. chr1,chr2,chr3+chr4)."
     doFilter: "Enable/disable Samtools filtering."
     doMarkDuplicates: "Enable/disable GATK4 MarkDuplicates."
-    doSplitNCigarReads: "Enable/disable GATK4 SplitNCigarReads."
-    doIndelRealignment: "Enable/disable GATK3 RealignerTargetCreator + IndelRealigner."
-    doBqsr: "Enable/disable GATK4 BQSR."
+    doBqsr: "Enable/disable GATK baseQualityScoreRecalibration"
     reference: "Path to reference file."
-    reference_genome: "reference genome of input sample"
+    outputFileNamePrefix: "Prefix of output file name"
+    reference_genome: "The reference genome version for input sample"
```

Comment:
Can this be updated to camel case? reference_genome --> refererenceGenome

Reply:
done, thanks!